README.md

```diff
@@ -15,39 +18,44 @@ NVIDIA Merlin is a scalable and GPU-accelerated solution, making it easy to buil
+HugeCTR is a GPU-accelerated training framework designed to scale large deep learning recommendation models by distributing training across multiple GPUs and nodes. HugeCTR contains optimized dataloaders with GPU-acceleration and provides strategies for scaling large embedding tables beyond available memory. With HugeCTR, you can:
 - scale embedding tables over multiple GPUs or nodes.
 - load a subset of an embedding table into the GPU in a coarse grained, on-demand manner during the training stage.
 
-**[Triton](https://github.com/triton-inference-server/server)**<br>
-NVTabular and HugeCTR both support the Triton Inference Server to provide GPU-accelerated inference. The Triton Inference Server is open-source inference serving software that can be used to simplify the deployment of trained AI models from any framework to production. With Triton, you can:
-- deploy NVTabular ETL workflows and trained deep learning models to production with a few lines of code.
-- deploy an ensemble of NVTabular ETL and trained deep learning models to ensure that the same data transformations are applied in production.
-- deploy models concurrently on GPUs to maximize utilization.
-- enable low latency inferencing in real time or batch inferencing to maximize GPU and CPU utilization. 
-- scale the production environment with Kubernetes for orchestration, metrics, and auto-scaling using a Docker container.
+**[Merlin Models](https://github.com/NVIDIA-Merlin/models)** [![PyPI version shields.io](https://img.shields.io/pypi/v/merlin-models.svg)](https://pypi.python.org/pypi/merlin-models/)<br>
+The Merlin Models library provides standard models for recommender systems with an aim for high quality implementations that range from classic machine learning models to highly-advanced deep learning models. It features:
+ - Performant dataloaders for Tensorflow, PyTorch and HugeCTR, accelerating ranking model training by up to 10x.
+ - Fast iteration of featuring engineering and model exploration by mapping datasets created by NVTabular into a model input layer automatically, allowing either to be changed without impacting the other.
+ - Connectable building blocks for common RecSys architectures, allowing for the creation of new models quickly and easily.
+
+**[Merlin Systems](https://github.com/NVIDIA-Merlin/systems)** [![PyPI version shields.io](https://img.shields.io/pypi/v/merlin-systems.svg)](https://pypi.python.org/pypi/merlin-systems/)<br>
+Merlin Systems provides tools for combining recommendation models with other elements of production recommender systems like feature stores, nearest neighbor search, and exploration strategies into end-to-end recommendation pipelines that can be served with Triton Inference Server.  It provides:
```

Comment:
Plan "A" is to just lowercase the initial word for each bullet.

Plan "B":

s/It provides:/With Merlin Systems, you can:/

- start with an integrated platform for serving recommendations built on Triton Inference Server.
- create graphs that define the end-to-end process for generating recommendations.
- benefit from existing integrations with existing tools that are commonly found in recommender system pipelines.